### ISSUE TITLE
[iOS] Use `RequestReviewAction` in favor of `SKStoreReviewController`

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#18902]

**Comment by Anderbr4** [comment]
Hello, if this has not been resolved already, could I work on this? I imagine it would be replacing the use of SKStoreReviewController in https://github.com/unoplatform/uno/blob/80872f26512983892bc54e7ef78b11305aca9206/src/Uno.UWP/Services/Store/StoreContext.UIKit.cs#L73 ?

**Comment by MartinZikmund** [comment]
Yes, it should be along these lines. Feel free to start working on it!

**Comment by CleanAmirHosseinDev** [comment]
Can you add more info?

**Comment by CleanAmirHosseinDev** [comment]
Interesting, can you explain further?
